dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassPathResultTest.java

```diff
@@ -115,21 +114,6 @@ public void testFormatDependencyPaths_threePathsForA() {
         actual);
   }
 
-  @Test
-  public void testFormatDependencyPaths_irrelevantJar() {
-    AnnotatedClassPath annotatedClassPath =
-        AnnotatedClassPath.fromMultimap(ImmutableListMultimap.of(jarA, dependencyPath_A));
-
-    ClassPathResult classPathResult = new ClassPathResult(annotatedClassPath, ImmutableSet.of());
-
-    try {
-      classPathResult.formatDependencyPaths(ImmutableList.of(jarB));
-      fail("The irrelevant JAR file should be invalidated.");
-    } catch (IllegalArgumentException expected) {
```

Comment:
With this PR, the method doesn't throw IllegalArgumentException any more.

Reply:
We have examples of capturing output, if you still want to test that the warning was printed: https://github.com/GoogleCloudPlatform/spring-cloud-gcp/blob/366b3d9530efda9b9b1e7dcd92d0284a38034c25/spring-cloud-gcp-samples/spring-cloud-gcp-integration-pubsub-sample/spring-cloud-gcp-integration-pubsub-sample-receiver/src/test/java/com/example/ReceiverIntegrationTest.java#L68-L71

Reply:
The warning is not critical message. I keep as it is.